web-components/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "cra-caddy-docker",
+  "homepage": "https://chexex.github.io/cra-caddy-docker/",
+  "private": true,
+  "scripts": {
+    "test": "CI=true react-scripts test --env=jsdom",
+    "test:coverage": "npm test --coverage",
```

Для второго задания здесь нужно будет только линтер прогонять (eslint .)

---

Прогнать линтер по шестой и седьмой строчке? Добавить в web-components/package.json что-то вроде этого, по аналогии с package.json вне web-components? Что-то другое?
```suggestion
"scripts": {
    ...
    "lint": "npm run lint:js && npm run lint:dup && npm run lint:css",
    "lint-fix": "eslint --fix",
...
} 
```

---

1. В веб компонентах нет реакта и реакт-скриптов :)
2. Этот файл для первого дз можно вообще не трогать
3. `npm run lint:js`, `npm run lint:dup`, `npm run lint:css` подразумевают, что у тебя описаны инструкции для `lint:js`, `lint:dup` и `lint:css`
```js
// ...
  "lint:js": ...
  "lint:dup": ...
  "lint:css": ...
// ...
```